laghos.cpp

```diff
@@ -296,7 +296,7 @@ int main(int argc, char *argv[])
     args.AddOption(&blast_energyFactor, "-bef", "--blastefactor", "Factor for scaling blast energy.");
     args.AddOption(&rom_paramID, "-rpar", "--romparam", "ROM offline parameter index.");
     args.AddOption(&romOptions.paramOffset, "-rparos", "--romparamoffset", "-no-rparos", "--no-romparamoffset",
-                   "Enable or disable parametric offset.");
+                   "Enable or disable parametric offset."); // TODO: redundant, remove after PR 98 and remove in regression tests
```

Comment:
Is this here just so regression tests will pass? Does it need to be removed immediately after PR approval?

Reply:
@dylan-copeland Yes it is kept here just to make sure the regression tests are passed, but do not appear anywhere in the main program. This line and `-rparos` options in the regression test scripts will be removed right after the PR approval.